contracts/test/BasicFDT.t.sol

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.6.11;
+
+import { MapleTest }        from "../../modules/maple-test/contracts/test.sol";
+import { ERC20, SafeMath }  from "../../modules/openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
+
+import { BasicFDT }         from "../BasicFDT.sol";
+import { IBasicFDT }        from "../interfaces/IBasicFDT.sol";
+
+import { Account }          from "./accounts/Account.sol";
+
+
+contract MockFDT is BasicFDT {
+
+    uint256 public fundsBalance;
+    uint256 public lastFundsBalance;
+
+    constructor(string memory name, string memory symbol) public BasicFDT(name, symbol) {}
+
+
+    function mint(address to, uint256 amt) public {
+        _mint(to, amt);
+    }
+
+    function burn(address account, uint256 amt) public {
+        _burn(account, amt);
+    }
+    function withdrawFunds() public virtual override {
+        uint256 withdrawableFunds = _prepareWithdraw();
+        fundsBalance = fundsBalance.sub(withdrawableFunds);
+        _updateFundsTokenBalance();
+    }
+
+    function _updateFundsTokenBalance() internal override returns (int256 delta) {
+        delta = int256(fundsBalance - lastFundsBalance);
+        lastFundsBalance = fundsBalance;
+    }
+
+    function increaseFundsReceived(uint256 amount) external{
+        fundsBalance = fundsBalance + amount;
+    }
+
+     function pointsCorrection_(address account) external view returns(int256) {
+        return pointsCorrection[account];
+    } 
+
+    function pointsPerShare_() external view returns(uint256) {
```

Comment:
What do you mean? Like creating a function that overrides the state variable:
```
function pointsPerShare() external view override returns (uint256) {
```
Because that particular syntax does not compile, and I think we would need to change the BasicFDT itself to work.